Code Review

matalkbt opened the pull request:
Was the program available on time? 
Yes
Is the program documented well enough for you to understand?
Yes, because there isn't really much code to look over. There is 2 tests and a really well designed layout. But no code for the actual program.
Does the program compile?
Yes! Everything compiles and builds perfectly.
The only thing I added was an empty DTO class. I think it would be a good next step to start creating your drink objects and working them into your project so that you can get the ball rolling.

3 Technical Concepts I learned:
Well... There wasn't any code, so I didn't learn much of anything unfortunately. I learned more from the other code review done on this project.

JoshuaGyau committed:
add basic dto class

brownadam010 committed:
my code review suggestions

brownadam010 committed:
my code review suggestions

richrn committed:
Changed orintations to fit screen better

matalkbt committed:
Code Review

matalkbt commented:
Just an FYI I think I messed up the push some how... So if you have a random DTO file in master, I apologize

mastroja commented:
> Just an FYI I think I messed up the push some how... So if you have a random DTO file in master, I apologize

Yeah, it looks like you pushed to the master branch. Please create your own review branch and push to that. It's cool though, I needed to add a DTO anyway lol

mastroja committed:
Update README.md

discospiff commented on app/src/main/java/com/example/KnowYouLimit/drinksDTO.kt:
```diff
@@ -0,0 +1,6 @@
+package com.example.KnowYouLimit
+
+data class drinksDTO(var name:String, var type:String ) {
```
Is this used anywhere?

discospiff commented:
It looks like the data class is already used by the team... so no further merge advice needed from me.

mastroja committed:
Merge pull request #1 from JoshuaGyau/brown5a7_codeReview1

Brown5a7 code review1

richrn committed:
Merge remote-tracking branch 'origin/master'

richrn committed:
Changed orientations  to fit screen better

richrn committed:
Added ui for intoxication meter. I used buttons for the numbers specified in the mockup.

mastroja committed:
logo creation

i dont really like this logo but its something for now

brownadam010 committed:
My code review 2 suggestions

mastroja committed:
Merge pull request #4 from mastroja/brown5a7-codeReview2Final

My code review 2 suggestions

mastroja committed:
a lot

ui is finished, page functionality is finished.
need to use dto and dao to fill in lists
add notification to update intoxication meter
allow for adding new drink function

mastroja committed:
Update README.md

mastroja committed:
Update README.md

mastroja committed:
add drink activity

forgot this one lol

mastroja committed:
Merge branch 'master' of https://github.com/mastroja/KnowYourLimit-MobileApp

mastroja committed:
dto added

dto added drink data stored in respective activities

mastroja committed:
popup

added popup notification, having trouble changing its visibilty on click

richrn committed:
Uploading listener with radio buttons

richrn committed:
Uploading listener with radio buttons

mastroja committed:
popup & intoxication meter

popup feedback now appears and disappears correctly, the users feedback then indicates when the arrow is on the intoxication meter

mastroja committed:
Merge branch 'master' of https://github.com/mastroja/KnowYourLimit-MobileApp

mastroja committed:
cleaning up code

cleann

mastroja committed:
arrow error fixed

fixed small issue

richrn committed:
Fixed Crashng issues with button click.
Fixed/added toast on button click
